controllers/selfnoderemediationconfig_controller.go

```diff
@@ -295,3 +299,24 @@ func (r *SelfNodeRemediationConfigReconciler) removeOldDsOnOperatorUpdate(ctx co
 	return nil
 
 }
+
+func (r *SelfNodeRemediationConfigReconciler) setCRsEnabledStatus(ctx context.Context) error {
+	crs := selfnoderemediationv1alpha1.SelfNodeRemediationList{}
+	if err := r.List(ctx, &crs); err != nil {
+		r.Log.Error(err, "Failed to list self node remediation CRs")
+		return err
+	}
+
+	for _, cr := range crs.Items {
+		//is disabled
+		if meta.IsStatusConditionTrue(cr.Status.Conditions, string(selfnoderemediationv1alpha1.DisabledConditionType)) {
```

Comment:
This might be a bit misleading. The comment says "enable", but we remove the condition.
We can actually set it to false instead, as "advertisement" for the new condition (If users start seeing the condition set to False, they'll know it _can_ be True)

Reply:
Feels to me like something of a personal preference. I agree that for some conditions it's clearer to have either a false or a true value.

Specifically I think that for this one though I think having no condition is better than disabled false, trying to explain my rational:
-  the general  assumption is that the operator is working so I  having a Disabled False condition to describe this basic assumption seems a bit confusing
- I think that in case we want to have the Disabled False condition we should probably set it from the get go otherwise we can create a confusion about the difference between  having  a Disabled False condition (after a deleted config is created) and not setting this condition (when operator starts).

Having said all that it's not something I feel too strongly about :sweat_smile: , so let me know if you still prefer it changed.

P.S in case we decide to use this condition all the time I think that Enabled is probably better than Disabled

Reply:
My point on having it always set (enabled or disabled) was mostly as advertisement, but otherwise I'm OK with removing it.
I was only a bit confused by the term "set enable status", when what the function does is removing the _disabled_ condition.

Reply:
NP, I'll improve the comment syntax